basics.js

```diff
@@ -1,24 +1,70 @@
 /* Create a `myName` variable and assign it a String value */
+var myName = "Dev League";
 
 /* Create a `person` variable and give it 2 properties,
  * `name`, assign it the same name as before,
  * as well as an `age` (number);
  */
+var person = {name: myName, age: 60};
 
 /* Create a function called `greet`,
  * it should take a 1 parameter, `name`
  * and it should print "Hello, my name is {name}"
  */
+function greet(name)
+  {
+    console.log('Hello, my name is ' + name);
+  }
 
+greet(person.name);
 /* Create a variable called `canDrive`,
  * if it should be true if your person object is at least 16 years old
  */
+function canDrive (age) {
+  if (age >= 16) {
+    return true;
+  }
+  else {
+    return false;
+  }
+}
+var canDrive = canDrive(person.age);
+console.log(canDrive);
```

Comment:
`console.log` is okay for testing output locally. But please delete before committing.